Source/Core/Core/IOS/ES/NandUtils.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Core/IOS/ES/Formats.h"
+
+namespace IOS
+{
+namespace ES
+{
+TMDReader FindImportTMD(u64 title_id);
+TMDReader FindInstalledTMD(u64 title_id);
+
+// Returns a vector of title IDs in the /title directory, without checking for TMDs at all.
```

Comment:
I'd avoid mentioning the specific type being returned, mainly because it's already evident from the signature, and if whatever is being returned needs to change, you don't need to update the comment.